Gems/PhysX/Code/Source/RigidBodyComponent.h

```diff
@@ -52,35 +54,34 @@ namespace PhysX
         static void GetProvidedServices(AZ::ComponentDescriptor::DependencyArrayType& provided)
         {
             provided.push_back(AZ_CRC_CE("PhysicsRigidBodyService"));
+            provided.push_back(AZ_CRC_CE("PhysicsDynamicRigidBodyService"));
         }
 
         static void GetIncompatibleServices(AZ::ComponentDescriptor::DependencyArrayType& incompatible)
         {
             incompatible.push_back(AZ_CRC_CE("PhysicsRigidBodyService"));
-            incompatible.push_back(AZ_CRC_CE("PhysicsStaticRigidBodyService"));
         }
 
         static void GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required)
         {
             required.push_back(AZ_CRC_CE("TransformService"));
         }
 
-        static void GetDependentServices(AZ::ComponentDescriptor::DependencyArrayType& dependent)
+        static void GetDependentServices([[maybe_unused]] AZ::ComponentDescriptor::DependencyArrayType& dependent)
         {
-            dependent.push_back(AZ_CRC_CE("PhysicsColliderService"));
         }
 
-        // RigidBodyRequests + WorldBodyRequests
+        // RigidBodyRequests + SimulatedBodyComponentRequests overrides ...
         void EnablePhysics() override;
         void DisablePhysics() override;
         bool IsPhysicsEnabled() const override;
 
         AzPhysics::SceneQueryHit RayCast(const AzPhysics::RayCastRequest& request) override;
         AZ::Aabb GetAabb() const override;
 
-        // RigidBodyRequests
+        // RigidBodyRequests overrides ...
         AZ::Vector3 GetCenterOfMassWorld() const override;
-        virtual AZ::Vector3 GetCenterOfMassLocal() const override;
+        AZ::Vector3 GetCenterOfMassLocal() const override;
```

Comment:
question: Is it worth grouping these with the above comment block now and then not repeating it here?

Reply:
It's already correctly grouped. It turns out that 2 buses (RigidBodyRequests and SimulatedBodyComponentRequests) have the same interface partially (probably something to improve in the future), so it's better to group it how it is now to make it clear what is common and what is unique from each bus, otherwise it would look like there are functions missing from one bus.

Reply:
Hmm I see, I just found the duplicated comment block confusing but I think it looks like it's fixed now